Dockerfile

```diff
@@ -11,12 +11,12 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-FROM golang
+FROM golang:1.15 as builder
 COPY . /go/src/github.com/ossf/scorecard
 WORKDIR /go/src/github.com/ossf/scorecard
 RUN [ "go", "build", "."]
 
-FROM gcr.io/cloud-builders/gsutil
+FROM gcr.io/distroless/base:nonroot
```

Comment:
Ah, I don't think this part is safe. We actually use gsutil in the nightly cron job from inside the container to upload results to GCS here: https://github.com/ossf/scorecard/blob/main/cron/cron.sh

We could either reimplement that part in Go or try to use a separate container.

Any other ideas?

Reply:
IMO we could have a separate Dockerfile just for the scorecard without using gsutil and publish that image for pulling down the scorecard. How does that sound?

Reply:
SGTM! We could move this one into the cron directory to make it more clear as well.

Reply:
Sounds good. Thanks